api/requests/get_batches.go

```diff
@@ -0,0 +1,15 @@
+package requests
+
+import ethCommon "github.com/ethereum/go-ethereum/common"
+
+// GetBatchesAPIRequest is an API request struct for getting batches
+type GetBatchesAPIRequest struct {
```

Comment:
2. This is a subjective comment: I would have called these structs `...Args` (instead of `...Request`) because they are the arguments to a function, and then inside the function, I would have called the variable `args` (instead of `request`).  This is very subjective so feel free to ignore it.